get-started/graph-database/graph-database.adoc

```diff
@@ -22,12 +22,9 @@ toc::[]
 
 == Overview
 
-Very simply, a graph database is a database designed for storing and retrieving data and the relationships among it.
-Rather than prioritizing a rigidly-formatted structure to force on the data and navigating the structure to analyze sets of data, a graph database directly connects each piece of data to those it is related to, elevating the relationships in the data model to equal importance with the data itself.
-
-++++
-<iframe width="680" height="425" src="https://www.youtube.com/embed/_D19h5s73Co" frameborder="0" allowfullscreen></iframe>
-++++
+Very simply, a graph database is a database designed to treat the relationships between data as equally important to the data itself.
+It is intended to hold data without constricting it to a pre-defined model.
+Instead, the data is stored like we first draw it out - showing how each individual entity connects with or is related to others.
```

Comment:
I think these opening sentences need to be clearer. The data model in Neo4j is still usually pre-defined. And the reader is probably unaware what sort of drawing you mean. A picture of a graph up here on the page (to make it obvious right away we're not talking about pie charts) might be good. 

Maybe just delete lines 26/27 and switch the sections "Why Graph Databases?" and "The Property Graph Model" so that graph databases are described in terms of the data model before their importance is explored